spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "takeaway"
+
+describe Takeaway do
+
+  let(:menu) { double }
+  subject(:takeaway) { described_class.new(menu) }
+
+  describe "menu" do
+
+    it "can check the menu" do
+      expect(takeaway.menu).to eq menu
+    end
+  end
+
+  describe "#place_order" do
+
+    context "when correct total given" do
+      it "adds a single item to the order" do
+        allow(menu).to receive(:dishes).and_return({ "spring rolls" => 2 })
+        takeaway.place_order("spring rolls", 3, 6)
```

Comment:
Refactoring how an order is represented would also make this test more readable (potentially).  At the moment there are two numbers being passed to `Takeaway#place_order` and I have to refer back to the code in order to work out what those numbers mean.